pom.xml

```diff
@@ -45,7 +45,8 @@
   <properties>
     <maven.assembly.version>2.2.1</maven.assembly.version>
     <hbase.version>0.92.1</hbase.version>
-    <cassandra.version>0.7.0</cassandra.version>
+    <cassandra.version>1.2.9</cassandra.version>
```

Comment:
I added the new binding as a separate class CassandraCQLClient and left the previous bindings untouched.  So, you can use those with pre-1.2 servers.  But this property is used in the maven build to pull in the right dependencies for the new CassandraCQLClient.  I don't know if the dependencies for the pre-1.2 version are still being pulled in with this change.

I suppose I could have created a "cassandracql.version" property?  Then both old and new jars would be pulled, but I'm skeptical about that working.  Some dependency jars could get clobbered with older or newer versions.  I might tend towards creating it as a separate binding completely, with its own directory if we need old and new to work side-by-side.

I'm not sure this answers your question.  The folks at Datastax told me that Cassandra CQL was the direction of the data store moving forward.  They did not recommend using the older binding.  I did not test with it.

Thanks for your work!  Unfortunately, I don't have access to the environment I had when I did this work, but I'll help as much as I can.